Week1/Homework/logtheremainder.js

```diff
@@ -0,0 +1,11 @@
+let x = 7;
+x = x % 3;
+console.log(x); // ('The value of x would be 1');
+
+let y = 21;
+y = y % 4; // ('The value of y would be 1')
+console.log(y);
+
+let z = 13;
+z = z % 2; // ('The value of y would be 1')
+console.log(z);
```

Comment:
👍 for logic but there seems to be a confusion in this question which I am checking currently with Noer.